src/main/java/tutoraid/storage/JsonAdaptedStudent.java

```diff
@@ -56,15 +57,15 @@ public JsonAdaptedStudent(Student source) {
         parentName = source.getParentName().fullName;
         parentPhone = source.getParentPhone().value;
         progressList = source.getProgressList().getAllProgressAsStringArrayList();
-        lessons = source.getLessons().getAllLessonNamesAsStringArrayList();
+        lessonNames = source.getLessons().getAllLessonNamesAsStringArrayList();
```

Comment:
Lesson names used to create InitialStudent. When Student is created, we will use Lesson instead.